DiscordBot/Commands/EventCommands.cs

```diff
@@ -1,9 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
+﻿#nullable enable
```

Comment:
You should be able to do this for the whole project, by adding it to the project file. Let me know if you have any issues with it 🙂